introduction.md

```diff
@@ -1,3 +1,11 @@
 I AM SIRIUSDARKZ
 
-MY FAVORITE CODESCHOOL PATH ARE Electives.
\ No newline at end of file
+MY FAVORITE CODESCHOOL PATH ARE Electives.
+
+*JAVA
+*JavaScript
+*CSS
+*HTML
+*GIT
+*PYTHON
+*RUBY
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?